### ISSUE TITLE
Accessibility: react-slider doesn't announce to assistive tech that it is disabled

### ISSUE THREAD

**Issue by moloko** [zillow/react-slider#267]
## Description
When using assistive technology such as a screen reader, this component does not indicate to the screen reader when it is disabled

Adding `aria-disabled` to the 'handle' elements should resolve this easily.

**Comment by moloko** [comment]
this is still an issue, please don't close

**Comment by kris-ellery** [comment]
Thank you for your patience. Merged and published in v2.0.5
